WordPress/Classes/ViewRelated/System/Bottom Sheet/BottomSheetContentViewController.swift

```diff
@@ -0,0 +1,113 @@
+import UIKit
+import SwiftUI
+import WordPressUI
+
+class BottomSheetContentViewController: UIViewController {
```

Comment:
This logic was extracted from `CompliancePopoverViewController`. I was going to implement the same class for the `Comment Moderations Options View` but then I thought why not implement a reusable component instead?